aws/resource_aws_api_gateway_authorizer.go

```diff
@@ -55,6 +56,11 @@ func resourceAwsApiGatewayAuthorizer() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"provider_arns": &schema.Schema{
+				Type:     schema.TypeList,
```

Comment:
Does the ordering of the values matter for the `provider_arns` attribute? If not, we should probably switch this to `schema.TypeSet` 👍